hatice-ecevit-2016400138/assignment1/Node.java

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Style: remove this blank line.
The way I think about blank lines is:

- does it mean something?
- does it make things easier to read?

For example, having a blank line between two blocks of code that represent "part 1" and "part 2" of the algorithm does have a meaning.
Another example, adding a blank line between the fields and the methods, or between methods makes the code more readable (you could do so below!).